lib/game_of_life_web/views/universe_view.ex

```diff
@@ -0,0 +1,13 @@
+defmodule GameOfLifeWeb.UniverseView do
+  use GameOfLifeWeb, :view
+
+  def render_cells(cells, {width, height}) do
+    Enum.map(0..(height - 1), fn y ->
+      content_tag(:div, class: "cell-row") do
+        Enum.map(0..(width - 1), fn x ->
+          GameOfLifeWeb.CellView.render_cell(Map.get(cells, {x, y}))
+        end)
+      end
+    end)
+  end
+end
```

Comment:
This is awesome, I don't think I ever saw `content_tag` like this.. I love that it takes a block, too